osc-dm-product-srv/src/models.py

```diff
@@ -46,6 +46,30 @@ class Artifact(BaseModel):
     service: Optional[str] = None
 
 
+class ProductGen(BaseModel):
+    outputDir: str
+    fileName: str
+    namespace: str
+    name: str
+    tags: Optional[str] = None
+    description: Optional[str] = None
+    url: Optional[str] = None
+    vendor: Optional[str] = None
+    model: Optional[str] = None
+
+class ArtifactGen(BaseModel):
+    outputDir: str
+    fileName: str
+    name: str
+    tags: Optional[str] = None
+    dataUrl: Optional[str] = None
+    description: Optional[str] = None
+    url: Optional[str] = None
+    vendor: Optional[str] = None
+    model: Optional[str] = None
+    artifactType: Optional[str] = None
+    host: Optional[str] = None
+
```

Comment:
use below format to include descriptions with your fields. Makes the auto-generated openapi documentation automatically contain that information. 

If no default value
```
latitude: float = Field(description="The latitude of the central point")
```

If default value
```
month: Optional[int] = Field(
        None, description="The month to retrieve data for")
```